zosia16/settings/__init__.py

```diff
@@ -1 +1,4 @@
 from .common import *
+import os
+
+GAPI_KEY = os.environ.get('GAPI_KEY')
```

Comment:
Trochę człowiek może się pogubić, że ma w inicie a potem w commonie puste a na koniec musi i tak to deva dołączyć :)